datafusion/core/tests/sqllogictests/test_files/type_coercion.slt

```diff
@@ -29,3 +29,21 @@ query B
 select 's' is not distinct from arrow_cast('s', 'LargeUtf8');
 ----
 true
+
+# date - interval
+query D
+select  '2023-05-01'::date - interval '1 month';
+----
+2023-04-01
+
+# date - interval
```

Comment:
this seems identical to the test above it (maybe it was supposed to be `date + interval`?)

Reply:
I copied twice🥲